libraries/AP_HAL_ChibiOS/hwdef/SuccexF4/hwdef.dat

```diff
@@ -132,5 +132,7 @@ define HAL_SPRAYER_ENABLED 0
 define AP_PARAM_MAX_EMBEDDED_PARAM 1024
 
 # minimal drivers to reduce flash usage
-include ../include/minimal.inc
+include ../include/minimize_fpv_osd.inc
 include ../include/no_bootloader_DFU.inc
```

Comment:
I suspect this can be removed

Reply:
why? its been there with no complaints...why reduce its space?

Reply:
removing the bootloader is again inconsistent and problematic for users, we should avoid it as much as possible - many of my builds do not have an accessible dfu button, removing the bootloader makes it impossible to upgrade. The fact that you have not had complaints means little here

Reply:
sorry you can easily upgrade firmware with is removed without touching the button...this option removes the incorporated bl image from the main firmware which is used only to flash update the bootlader...

Reply:
Sorry, I do not agree - we spent ages agreeing on a minimal set of features and removing the bootloader was not one of them. If you are going to use the minimal set then please use the minimal set that we agreed on, not the one we didn't.